Tests/Translation/Extractor/File/FormExtractorTest.php

```diff
@@ -247,11 +247,11 @@ private function extract($file)
         $file = new \SplFileInfo($file);
 
         $lexer = new Lexer();
-        if(class_exists('PhpParser\ParserFactory')) {
+        if (class_exists('PhpParser\ParserFactory')) {
             $factory = new ParserFactory();
-            $parser = $factory->create(ParserFactory::PREFER_PHP7,$lexer);
+            $parser = $factory->create(ParserFactory::PREFER_PHP7, $lexer);
         } else {
-            $parser = new \PHPParser_Parser($lexer);
+            $parser = new Parser($lexer);
```

Comment:
The \PHPParser_Parser is an old alias it had. There is a PhpParser\Parser class in 1.4.x and a PhpParser\Parser interface in 2.0. but 2.0 uses the if above with the Factory. Its just cleaner.